docs/products/kafka/howto/connect-with-command-line.rst

```diff
@@ -0,0 +1,64 @@
+Connect to Aiven for Apache Kafka® with command line tools
+==========================================================
+
+These examples show how to send messages to and receive messages from an Aiven for Apache Kafka® service using command line tools.
+
+Pre-requisites
+--------------
+
+``kafka-console-producer`` and ``kafka-console-consumer`` are part of the Apache Kafka® toolbox included with the `open source Apache Kafka® code <https://kafka.apache.org/downloads>`_. Follow
+`the guide to set up properties to use the Apache Kafka toolbox <kafka-tools-config-file.html>`_
+For ``kafka-avro-console-producer`` follow the installation instructions in `its GitHub repository <https://github.com/confluentinc/schema-registry>`_.
+
+Variables
+---------
+
+========================     ===========================================================================================
+Variable                     Description
+========================     ===========================================================================================
+``HOST``                     Host name for the connection
+``PORT``                     Port number to use for the Kafka service
+``CONFIGURATION_PATH``       Path to your configuration file `for Apache Kafka toolbox <kafka-tools-config-file.html>`_.
+``SCHEMA_REGISTRY_HOST``     Host name for your schema registry
+``SCHEMA_REGISTRY_PORT``     Port number for your schema registry
+``SCHEMA_REGISTRY_USER``     User name for your schema registry
+``SCHEMA_REGISTRY_PWD``      Password for your schema registry
+========================     ===========================================================================================
+
+Produce messages
+-----------------
+
+With ``kafka-console-producer`` you can send multiple messages into your topic.
+
+.. code::
+
+    kafka-console-producer --broker-list {HOST}:{PORT} \
+      --topic target-topic \
+      --producer.config {CONFIGURATION_PATH}
+
+Produce messages with schema
+----------------------------
+
+With ``kafka-avro-console-producer`` you can include the schema by connecting to your schema registry
```

Comment:
Again, it's worth saying that this reads from stdin, and maybe giving a very simple example - but we don't really want to show setting up a schema and so on, so it's not worth going into any detail, I think. The documentation for this command line tool appears to be at https://docs.confluent.io/platform/current/tutorials/examples/clients/docs/kafka-commands.html#consume-avro-records and it *might* be worth linking to.

Reply:
Good idea. I added the examples and the docs. The example will work with the provided schema through.